ts/output/chtml/fonts/tex/normal.ts

```diff
@@ -153,10 +153,10 @@ export const normal: CHTMLCharMap = AddCSS(font, {
     0x203E: {c: '\\2C9'},
     0x2044: {c: '/'},
     0x2057: {c: '\\2032\\2032\\2032\\2032'},
+    0x2060: {c: ''},
     0x2061: {c: ''},
     0x2062: {c: ''},
     0x2063: {c: ''},
-    0x2064: {c: ''},
```

Comment:
What's the reason for removing invisible plus?

Reply:
Sorry, I misread my Unicode chart.  I'll put it back.